src/sql/src/session/vars/definitions.rs

```diff
@@ -1813,7 +1813,7 @@ feature_flags!(
     {
         name: enable_explain_pushdown,
         desc: "EXPLAIN FILTER PUSHDOWN",
-        default: false,
+        default: true,
```

Comment:
@ggevay / @mgree

Reply:
This is fine. I think we could even completely remove this flag. cc @bkirwi

Reply:
I'm comfortable with that!

Reply:
Created an issue just so we remember to remove it at some point! https://github.com/MaterializeInc/database-issues/issues/8740